examples/stella_embedding/create_distill_data.py

```diff
@@ -0,0 +1,116 @@
+
+
+
+import tqdm
+import json
+from transformers import AutoTokenizer
+import sys
+from tqdm.autonotebook import trange
+import torch
+import numpy as np
+
+
+
+from sentence_transformers import SentenceTransformer
+
+
+def get_detailed_instruct(task_description: str, query: str) -> str:
+    return f'Instruct: {task_description}\nQuery: {query}'
+
+
+# Each query must come with a one-sentence instruction that describes the task
+task = 'Given a web search query, retrieve relevant passages that answer the query'
+
+def get_train_data(train_data_path):
+    train_data = []
+    train_data_text = []
+    dic = set()
+    with open(train_data_path) as f:
+        for line in tqdm.tqdm(f):
+            data_dic=json.loads(line.strip())
```

Comment:
`data_dict=json.loads(line)`

`json.loads` does not need strip

Complete variable names will never be an error